bindings/jupyter-modules/jupyter-ma-causal/src/containers/index.js

```diff
@@ -65,7 +70,9 @@ class App extends Component {
               <React.Fragment key={d.name}>
                 {['treatment', 'control'].map(groupName => (
                   <div key={groupName}>
-                    <div>{`${d.name} ${groupName} group`}</div>
+                    <div style={{paddingLeft}}>{`${
+                      d.name
+                    } ${groupName} group`}</div>
```

Comment:
The code format looks pretty weird... are you sure the linter is on?

Reply:
Yup, this style is produced by the linter, -.-